src/HotChocolate/Core/src/Types/Types/Descriptors/Conventions/DefaultTypeInspector.cs

```diff
@@ -41,18 +45,45 @@ public DefaultTypeInspector(bool ignoreRequiredAttribute = false)
     public bool IgnoreRequiredAttribute { get; protected set; }
 
     /// <inheritdoc />
-    public virtual IEnumerable<MemberInfo> GetMembers(Type type)
-        => GetMembers(type, false);
-
-    /// <inheritdoc />
-    public virtual IEnumerable<MemberInfo> GetMembers(Type type, bool includeIgnored)
+    public ReadOnlySpan<MemberInfo> GetMembers(
```

Comment:
Hello @michaelstaib, 
Due to exposing some domain entities to graphql schema, I used to override this method to exclude some properties with type `MediatR.INotification` like this

```c#
public class IgnoreTypeInspector : DefaultTypeInspector
    {
        public override IEnumerable<MemberInfo> GetMembers(Type type, bool includeIgnored)
        {

            return base.GetMembers(type, includeIgnored)
                .Where(e => e.Name != "DomainEvents" && e.Name != "BehaviourEvents");
            // This solves the error:
            // Interface `INotification` has no fields declared. (HotChocolate.Types.InterfaceType<MediatR.INotification>)
            // when inheriting from Entity and exposing it to HC
        }
    }
```

no that it is no longer `virtual` how can I workaround it. 

Thank you.